VRDR/Connectathon.cs

```diff
@@ -106,12 +106,12 @@ public static DeathRecord TwilaHilty()
             ije.AUTOP = "N";
             ije.AUTOPF = "X";
             ije.TOBAC = "U";
-            ije.PREG = "2";
+            ije.PREG = "8";//2";
             ije.PREG_BYPASS = "0";
             ije.CERTL = "D";
-            ije.SUR_MO = "01";
-            ije.SUR_DY = "10";
-            ije.SUR_YR = "2022";
+            ije.SUR_MO = null;
```

Comment:
You don't need to set to null, just remove these lines so it remains the default value